game.py

```diff
@@ -295,7 +340,20 @@ def _ask_the_question(self):
                         image=self._opts[i].type))
                 self._opts[i].set_layer(100)
         elif self._game == 1:
-            self._set_label(_('Recall which image was not shown.'))
+            text = []
+            text.append("¿")
+            text.append("    recall   ")
+            text.append(" not shown ")
+            text.append("   image   ")
+            text.append("?")
```

Comment:
```suggestion
            text = [
                "¿",
                "    recall   ",
                " not shown ",
                "   image   ",
                "?"
            ]
```

`list.append` uses a bit of extra resources which can be avoided. See [this](https://stackoverflow.com/q/14446128) for some useful information.